lib/installed-packages-panel.js

```diff
@@ -42,7 +42,7 @@ export default class InstalledPackagesPanel extends CollapsibleSectionPanel {
     )
     this.subscriptions.add(
       this.packageManager.on('package-install-failed theme-install-failed package-uninstall-failed theme-uninstall-failed package-update-failed theme-update-failed', ({pack, error}) => {
```

Comment:
Also this listens to too many events. `theme-install-failed`, `theme-uninstall-failed` and `theme-update-failed` should be removed. `theme-update-failed` then added to the themes panel. Also both themes and packages panel should also log to the console